django/db/backends/base/features.py

```diff
@@ -227,6 +227,17 @@ class BaseDatabaseFeatures:
     supports_select_difference = True
     supports_slicing_ordering_in_compound = False
 
+    # Window expressions (aggregate OVER (expression)), currently only
+    # supported it in PostgreSQL and Oracle
+    supports_over_clause = False
+
+    # Not every window function is supported in an OVER-clause
+    supported_window_functions = []
+
+    # Not every aggregate function is necessarily supported in every database
+    # backend
+    supported_window_aggregates = []
```

Comment:
If I'm not mistaken these aren't checked anywhere yet.

I'm wondering will this style make it harder than necessary to write new window functions - if you write a new window function for your own use, will you need to then also patch the backend?